java/src/test/java/ai/rapids/cudf/DecimalColumnVectorTest.java

```diff
@@ -0,0 +1,187 @@
+/*
+ *  Copyright (c) 2019-2020, NVIDIA CORPORATION.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package ai.rapids.cudf;
+
+import ai.rapids.cudf.HostColumnVector.Builder;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class DecimalColumnVectorTest extends CudfTestBase {
+  private final Random rdSeed = new Random();
```

Comment:
Unit tests that are not deterministic are infuriating to debug if they fail sporadically, because we don't know what values they used to generate the failure.  Always use a deterministic seed when generating random values in tests so the unit tests are deterministic.

Reply:
Fixed.